GG/GG/Font.h

```diff
@@ -206,6 +209,14 @@ class GG_API Font
 
         virtual ~TextElement(); ///< Virtual dtor.
 
+        /** Rebind to \p whole_text, by binding the SubString data member text to \p whole_text.
+
+            Binding to a new \p whole_text is very fast compared to re-parsing the entire \p
+            whole_text and allows TextElements of TextElementType TEXT to be changed quickly if it
+            is known that the parse would be the same.
```

Comment:
Can you give an example when this would apply?

My initial thought was adding / removing underlines, but in that case the extra <u> and </u> would presumably move the index in the whole_text string that each TextElement should point to.

Perhaps colour changes, eg. for mouseovers, but then wouldn't the number of characters in the numbers in the rgba tag possibly change?